src/lesson7/javaBeginnerChapter2/task25/ATM.java

```diff
@@ -0,0 +1,59 @@
+package lesson7.javaBeginnerChapter2.task25;
+
+public class ATM {
+    private int twentyDollars;
+    private int fiftyDollars;
+    private int hundredDollars;
+
+    public ATM(int twentyDollars, int fiftyDollars, int hundredDollars) {
+        this.twentyDollars = twentyDollars;
+        this.fiftyDollars = fiftyDollars;
+        this.hundredDollars = hundredDollars;
+    }
+
+    public void addMoney(int twentyDollars, int fiftyDollars, int hundredDollars) {
+        this.twentyDollars += twentyDollars;
+        this.fiftyDollars += fiftyDollars;
+        this.hundredDollars += hundredDollars;
+    }
+
+    public Boolean withdrawMoney(int twentyDollars, int fiftyDollars, int hundredDollars) {
```

Comment:
Method should receive one argument. And you need to write logic for appropriate withdraw

Reply:
fixed